docs/current/sailbot_workspace/workflow.md

```diff
@@ -94,25 +123,32 @@ launch configueration.
 
 If you are having some trouble running our software, here are some things you can try:
 
-ROS:
-
-- Run the `clean` VS Code task to clean your build
-- Run the `purge` VS Code task to delete all generated files in the workspace
-
-Dev Container:
-
-- Run the `Dev Containers: Rebuild Container` command in the
-  [VS Code command palette](https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette){target=_blank}
-  to rebuild the Dev Container
-
-VS Code:
-
-- Close and reopen the VS Code
-- Update VS Code and its extensions
-
-Docker:
-
-- Run `docker system prune` to remove all unused containers, networks, images (both dangling and unreferenced)
-    - Add `--all` to additionally remove unused images
-    - Add `--volumes` to additionally remove volumes
-    - Run `docker rmi -f $(docker images -aq)` to remove all images
+- Delete generated files
+    - Run the `clean` task to clean your build
+    - Run the `purge` task to delete all generated files
+- Rebuild the Dev Container: run the `Dev Containers: Rebuild Container` command
+- Reload VS Code: run the `Developer: Reload Window` command
+- Delete Docker files
+    - Run `docker system prune` to remove all unused containers, networks, images (both dangling and unreferenced)
+        - Add `--all` to additionally remove unused images (don't have a container associated with them)
+        - Add `--volumes` to additionally remove volumes (makes Bash history and ROS logs persist across containers)
+        - Run `docker rmi -f $(docker images -aq)` to remove all images
+
+    ??? tip "Running Docker CLI commands on Windows"
```

Comment:
Should it always be done instead of it being a tip? I remember that the tasks where only present when I ran the workspace through ubuntu terminal first.